server/src/modules/genres/dto/create-genre.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString, MaxLength, MinLength } from 'class-validator';
+
+export class CreateGenreDto {
+  @IsNotEmpty()
+  @IsString()
+  @MinLength(2)
+  @MaxLength(32)
+  @ApiProperty({
+    example: 'phonk',
```

Comment:
add option uniqueItems in Swagger Api Property (according to entity name unique field)